README.md

```diff
@@ -1,2 +1,38 @@
-# use-is-mounted-ref
-Returns a ref with a current value of true if mounted
+# useIsMountedRef
+Returns a `current` value of true when a component is mounted.
+
+## Motivation
+Useful when working with server side rendering. Comparable to `componentDidMount` in a class-based component. Does __not__ trigger a rerender.
+
+## Installation
+
+```
+yarn add @kaliber/use-is-mounted-ref
+```
+
+## Usage
+```jsx
+import { useIsMountedRef } from '@kaliber/use-is-mounted-ref'
+
+export default function App() {
+  const [, setMounted] = React.useState(false)
```

Comment:
Wat je met setMounted doet is eigenlijk niet het 'mounten' van het component, maar het rerenderen ervan zodat de nieuwe waarde wordt opgepakt. Zou het wellicht duidelijker zijn om iets van 
```
`const forceRender = useForceRender()`
```
en dan die functie te gebruiken?
te doen

Reply:
```
set

...

// calliing set with a new value triggers a rerender
```